actions/playerActions.js

```diff
@@ -29,15 +30,18 @@ const getPlayerError = (payload) => ({
 export const getPlayer = (accountId, host = HOST_URL) => (dispatch) => {
   dispatch(getPlayerRequest());
   dispatch(getPlayerMatchesRequest());
+  dispatch(getPlayerHeroesRequest());
```

Comment:
no it doesn't actually make a request, it just tells that piece of state that it's in a loading state (because player request is happening here).